AD-reviews/review-ietf-dnssd-prireq-04-genart-lc-sparks-2020-02-07.txt

```diff
@@ -0,0 +1,41 @@
+* review-ietf-dnssd-prireq-04-genart-lc-sparks-2020-02-07
+
+This is a combined genart and secdir last-call review.
+Please treat these comments just like any other last call comments.
+
+Document: draft-ietf-dnssd-prireq-04
+Reviewer: Robert Sparks
+Review Date: 2020-02-07
+IETF LC End Date: 2020-02-12
+IESG Telechat date: Not scheduled for a telechat
+
+Summary: Ready (but with nits) for publication as an Informational RFC
+
+This document provides a set of high-level requirements for a DNS-SD
+privacy exptension, and discussion motivating those requirements.
+
+Comment:
+It might be good to call out in the discussion that while it is intended
+to be thorough, it's not possible to be exhaustive.
+
+Nits (editorial, in document order):
+
+The last sentence of the first paragraph of the introduction is complex.
+Consider breaking it apart.
+
+In the introduction at "When analyzing these scenarios in Section 3.2",
+did you mean Section 3.1?
+
+In the first sentence of 3.2 at "the scenarios in Section 2", did you
+mean Section 3.1?
+
+At the first sentence in 3.4.4, at "online" did you mean "on-link"?
+
+The statement in the second paragraph of section 4 is perhaps too strong.
+Consider changing "will lead" to "are intended to lead".
+
+The item numbering in sections 4.1 and 4.2 are messsed up.
+
+The intent of the next to last paragraph in 4.1 and the last paragraph in 4.2
+could be made more clear. I suggest something like: "When listing and resolving
+services in current DNS-SD deployments".
```

Comment:
Why do you need to add a file in the repo for that? We are not using that text directly in the production! My customary way of handling these is to add an issue (use the new issue button in Github), cut and paste the review as the first issue comment, and then use that for tracking. If all goes well, the issue should be closed by the PR.